src/ZPI.API/DTOs/AssetDto.cs

```diff
@@ -10,5 +10,8 @@ public sealed record AssetDto(
     string FriendlyName,
     
     [property: JsonProperty(Required = Required.Always)]
-    string Category
+    string Category,
+
+    [property: JsonProperty(Required = Required.AllowNull)]
+    string Symbol
```

Comment:
Zmien z `string` na `string?` wtedy mowisz ze to pole jest nullowalne

Reply:
wrzuciłem też w modelu